sigs.yaml

```
@@ -1204,13 +1204,21 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes-sigs/contributor-playground/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/community/master/mentoring/OWNERS
     meetings:
-    - description: sig-contribex mentoring subproject meeting
-      day: Friday
-      time: "10:00"
+    - description: sig-contribex mentoring subproject meeting (APAC Friendly Time)
+      day: Monday
+      time: "20:30"
       tz: PT
       frequency: biweekly
       url: https://zoom.us/j/739807514
```

```suggestion
      url: https://zoom.us/j/94884623370
```

They both have new zoom IDs so they come across to youtube correctly